Automattic/wp-calypso
# Don't show the action buttons in the streams activity cards

#### Changes proposed in this Pull Request

Show the action buttons (Download/Restore) in the rewindable streams activity cards and don't show it in the content items for that stream.

Before:
![image](https://user-images.githubusercontent.com/9832440/82205022-c8b9f780-98fd-11ea-85fa-c551092ced7a.png)

After:
![image](https://user-images.githubusercontent.com/9832440/82205211-1f273600-98fe-11ea-9400-690eafa2b26e.png)

Before:
![image](https://user-images.githubusercontent.com/9832440/82205356-61e90e00-98fe-11ea-966b-3ba71de2657f.png)

After:
![image](https://user-images.githubusercontent.com/9832440/82205415-7cbb8280-98fe-11ea-9a04-36d1f222f69d.png)

**More design information, on Figma:** https://www.figma.com/file/r9c7MPvvxT8QLLP8qr25Hc/Jetpack-Cloud-Style-Guide?node-id=1242%3A0

#### Testing instructions

- Apply the changes
- Select a site with Realtime Backup subscription
- Find a day with a stream of media upload, and check if you can download/restore your site to that point check. Then check the activity card content (the items) you can't download/restore each item because it's in a group of activities.
